stingray/pulse/pulsar.py

```diff
@@ -355,8 +355,8 @@ def fold_detection_level(nbin, epsilon=0.01, ntrial=1):
     if ntrial > 1:
         simon("fold: The treatment of ntrial is very rough. Use with caution")
     from scipy import stats
```

Comment:
Why are we importing scipy from within a function? Isn't that against our developer guidelines? :)

Reply:
Oh. (blush)